src/Orleans.Core.Abstractions/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,31 @@
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("Orleans")]
+[assembly: InternalsVisibleTo("ClientGenerator")]
+[assembly: InternalsVisibleTo("OrleansCounterControl")]
+[assembly: InternalsVisibleTo("OrleansTelemetryConsumers.Counters")]
+[assembly: InternalsVisibleTo("OrleansCodeGenerator")]
+[assembly: InternalsVisibleTo("OrleansRuntime")]
+[assembly: InternalsVisibleTo("OrleansProviders")]
+[assembly: InternalsVisibleTo("OrleansHost")]
+[assembly: InternalsVisibleTo("OrleansAzureUtils")]
+[assembly: InternalsVisibleTo("OrleansAWSUtils")]
+[assembly: InternalsVisibleTo("OrleansManager")]
+[assembly: InternalsVisibleTo("OrleansTestingHost")]
+[assembly: InternalsVisibleTo("LoadTestGrains")]
+[assembly: InternalsVisibleTo("UnitTests")]
+[assembly: InternalsVisibleTo("UnitTestGrainInterfaces")]
+[assembly: InternalsVisibleTo("UnitTestGrains")]
+[assembly: InternalsVisibleTo("TesterInternal")]
+[assembly: InternalsVisibleTo("TestInternalGrainInterfaces")]
+[assembly: InternalsVisibleTo("OrleansBenchmarks")]
+[assembly: InternalsVisibleTo("TestInternalGrains")]
+[assembly: InternalsVisibleTo("NonSilo.Tests")]
+[assembly: InternalsVisibleTo("TestExtensions")]
+[assembly: InternalsVisibleTo("Tester.AzureUtils")]
+[assembly: InternalsVisibleTo("Tester")]
+[assembly: InternalsVisibleTo("DefaultCluster.Tests")]
+[assembly: InternalsVisibleTo("Tester.SQLUtils")]
+[assembly: InternalsVisibleTo("Tester.ZooKeeperUtils")]
+[assembly: InternalsVisibleTo("AWSUtils.Tests")]
+[assembly: InternalsVisibleTo("GoogleUtils.Tests")]
```

Comment:
Let's not have any InternalsVisibleTo in abstraction. If for some practical reason we temporarily need 1 or 2, let's add those explicitly, but not all by default. Ideally by the time we release 2.0 we'll have none

Reply:
.I agree.

But as we discussed weeks ago, not having internal to at least Orleans and OrleansRuntime will be almost impossible (GrainId...)